pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -7,19 +7,19 @@ toc::[]
 
 With {pipelines-title}, you can create a customized CI/CD solution to build, test, and deploy your application.
 
-To create a full-fledged, self-serving CI/CD Pipeline for an application, you must perform the following tasks:
+To create a full-fledged, self-serving CI/CD pipeline for an application, you must perform the following tasks:
 
-* Create custom Tasks, or install existing reusable Tasks.
-* Create and define the delivery Pipeline for your application.
-* Create a PersistentVolumeClaim attached to the Workspace to provide the volume or filesystem for Pipeline execution.
-* Create a PipelineRun to instantiate and invoke the Pipeline.
-* Add Triggers to capture any events in the source repository.
+* Create custom tasks, or install existing reusable tasks.
+* Create and define the delivery pipeline for your application.
+* Provide a volume claim template, which creates a persistent volume claim attached to a workspace, or create a persistent volume claim. This persistent volume claim provides the storage volume or filesystem for the pipeline execution.
```

Comment:
s/attached/that is attached/

Also, I wonder if this either or might be a candidate for a second level of bullets?

> * Provide a volume with one of the following approaches
> ** By specifying a volume claim template, which creates...
> ** By specifying a PVC.

But with better mechanics.

And which _this_ applies for: _This persistent volume claim provides the storage volume or filesystem for the pipeline execution._ Is this applicable to only the PVC? If so, the 2 bullet approach might make it clear.

Just a thought.

Reply:
I liked the idea of nested bullets, I have played around with the words a bit, PTAL.